### PR TITLE
add issue-links for stack-overflow/discord

### LINKED ISSUE #6928: Provide stackexchange or github discussions q&a
# Feature Request

## Motivation

The "new question" template suggests

> Please contact the development team on [Discord](https://discord.gg/aptoslabs)

Not everyone likes the medium "chat".

## Pitch

Provide stackexchange or github discussions q&a

### PR DESCRIPTION
**Pull request by abebeos**
closes #6928
### Description

- Drop question template
- Add links to Stack ~Exchange~ Overflow and Discord

### Test Plan

- on a fork repo, merge the change to the default branch, the select "New Issue"

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1 +1,8 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Questions and Help (on Stack Exchange)
```
Comment (clay-aptos) [inline .github/ISSUE_TEMPLATE/config.yml:3]: on Stack Overflow

**Comment by clay-aptos** [comment]
Thanks for the change. I am adding more reviewers and have left one comment.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (abebeos) [reply .github/ISSUE_TEMPLATE/config.yml:3]: Oh my...

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1 +1,8 @@
 blank_issues_enabled: false
```
Comment (davidiw) [inline .github/ISSUE_TEMPLATE/config.yml:1]:
```suggestion
blank_issues_enabled: true
```

**Review by davidiw — APPROVED** [review]

**Comment by davidiw** [comment]
Guess we'll see what this looks like....

**Pull request merged**